src/Form/DataTransformer/TraversableToCollectionTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Form\DataTransformer;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+trait TraversableToCollectionTrait
```

Comment:
Do we need a trait for this? Sounds more like a static helper class.

Reply:
Should we move this under `Sonata\AdminBundle\Util\` namespace?